test/integration_new/object/folder_itest.py

```diff
@@ -0,0 +1,215 @@
+from datetime import datetime
+
+import pytest
+import hashlib
+import os
+
+from pytest_lazyfixture import lazy_fixture
+
+from boxsdk.exception import BoxAPIException
+from boxsdk.object.collaboration import CollaborationRole
+from test.integration_new.context_managers.box_metadata_template import BoxTestMetadataTemplate
+from test.integration_new.context_managers.box_test_group import BoxTestGroup
+from test.integration_new.context_managers.box_test_user import BoxTestUser
+from test.integration_new.context_managers.box_test_web_link import BoxTestWebLink
+from test.integration_new import CLIENT
+from test.integration_new import util
+from test.integration_new.context_managers.box_test_file import BoxTestFile
+from test.integration_new.context_managers.box_test_folder import BoxTestFolder
+
+FOLDER_TESTS_DIRECTORY_NAME = 'folder-integration-tests'
+
+
+@pytest.fixture(scope="module", autouse=True)
+def parent_folder():
+    with BoxTestFolder(name=f'{FOLDER_TESTS_DIRECTORY_NAME} {datetime.now()}') as folder:
+        yield folder
+
+
+@pytest.fixture(scope="module")
+def user():
+    with BoxTestUser(login=f'{util.random_name()}@box.com') as user:
+        yield user
+
+
+@pytest.fixture(scope="module")
+def group():
+    with BoxTestGroup() as group:
+        yield group
+
+
+def test_preflight_check(parent_folder):
+    file_size = 1213
+    accelerator_url = parent_folder.preflight_check(file_size, util.random_name())
+
+    assert accelerator_url
+
+
+def test_manual_chunked_upload(parent_folder, large_file_name, large_file_path):
+    total_size = os.stat(large_file_path).st_size
```

Comment:
In .net and Java we are generating large file on the spot not to keep it in the source code. Maybe you can do the same. Why someone need to clone >20MB file just to run integration test?